src/infra/sequelize/queries/frise/getGarantiesFinancièresDTO.integration.ts

```diff
@@ -77,69 +77,148 @@ describe(`Requête getGarantiesFinancièresDTO`, () => {
           statut: 'en retard',
           date: dateLimiteDépassée.getTime(),
           variant: 'admin',
+          actionPossible: 'enregistrer',
         });
       });
     });
 
     describe(`Retourner les données de GF en attente`, () => {
+      // role porteur
       it(`Etant donné un projet soumis à garanties financières
-          Et dont les GF sont en attente
-          Alors un GarantiesFinancièresDTO devrait être retourné avec un statut 'en attente'`, async () => {
+          Et dont les GF non soumises à la candidature sont 'en attente'
+          Si le l'utilisateur a le rôle 'porteur-projet'
+          Alors un GarantiesFinancièresDTO devrait être retourné avec : 
+          - un statut 'en attente'
+          - l'action possible 'soumettre'`, async () => {
+        const user = { role: 'porteur-projet' } as User;
         const garantiesFinancières = {
           statut: 'en attente',
           soumisesALaCandidature: false,
           dateLimiteEnvoi,
-          envoyéesPar: new UniqueEntityID().toString(),
+          envoyéesPar: null,
           dateConstitution: null,
-          dateEchéance: null,
           validéesPar: null,
+          dateEchéance: null,
           type: null,
         } as const;
 
         const résultat = await getGarantiesFinancièresDTO({
           garantiesFinancières,
-          user: utilisateurAutorisé,
+          user,
         });
 
         expect(résultat).toEqual({
           type: 'garanties-financières',
           statut: 'en attente',
           date: dateLimiteEnvoi.getTime(),
-          variant: 'admin',
+          variant: user.role,
+          actionPossible: 'soumettre',
         });
       });
 
-      it(`Etant donné un projet soumis à garanties financières,
-  et dont les GF sont en attente,
-  et pour lesquelles le type de la date d'échéance sont connus,
-  alors la requête getGFItemProps devrait retourner un GarantiesFinancièresDTO 
-  avec un statut 'en attente'`, async () => {
-        const dateEchéance = new Date('2025-01-01');
+      it(`Etant donné un projet soumis à garanties financières
+          Et dont les GF soumises à la candidature sont 'en attente'
+          Si le l'utilisateur a le rôle 'porteur-projet'
+          Alors un GarantiesFinancièresDTO devrait être retourné : 
+          - avec un statut 'en attente'
+          - sans d'action possible 'soumettre'`, async () => {
```

Comment:
c'est bizarre comme phrase non ? 

```suggestion
          - sans action possible`, async () => {
```